scout/server/app.py

```diff
@@ -131,6 +132,7 @@ def register_blueprints(app):
     app.register_blueprint(api.api_bp)
     app.register_blueprint(alignviewers.alignviewers_bp)
     app.register_blueprint(phenotypes.hpo_bp)
+    app.register_blueprint(diagnoses.omim_bp)
```

Comment:
Could you add a link to this page in the navbar?

Reply:
Good idea! I need to create a page dedicated to all omim terms before, because the one I made is for one term only!